utils/test/src/main/kotlin/Extensions.kt

```diff
@@ -43,5 +44,9 @@ fun TestConfiguration.getResource(name: String) = checkNotNull(javaClass.getReso
 
 fun TestConfiguration.readResource(name: String) = getResource(name).readText()
 
+fun TestConfiguration.readOrtResult(name: String) =
+    FileFormat.forExtension(name.substringAfterLast('.')).mapper
+        .readValue<OrtResult>(patchExpectedResult(readResource(name)))
```

Comment:
While it is consistent that these are extensions of `TestConfiguration`, should we make these non-extension top level functions ? e.g. `fun readOrtResult()`, because then it is possible to call them from any top / file level utility functions.

Reply:
That's not possible for anything that accesses resources because a class context is required.

Reply:
Hm, shouldn't the following, for example, work? `object {}.javaClass.getResource(path)?.readText()`

Reply:
Yes, but I consider the temporary creation of that "fake object" to be less nice than just making use of the `TestConfiguration` context that we already have.

Reply:
I might need to reconsider this during the migration once there is a need to call this from a top-level function, but right now I prefer the current solution.

Reply:
> Yes, but I consider the temporary creation of that "fake object" to be less nice

It's not necessary that the fake object is only temporary.